builder/rootfs/etc/confd/templates/check-repos

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env bash
 
+set -x
+
 export ETCD=${ETCD:-$HOST:4001}
 
 cd $(dirname $0) # absolute path
 
-for repo in *.git
+for repo in $(ls | grep .git)
```

Comment:
I actually found issues with this. If no git repositories existed, a non-zero error was raised. Instead I just wanted the number of repos so I went with the tried and true approach.